### DISCUSSION TITLE
Disable device exposure

### DISCUSSION THREAD

**Discussion by OmriMulti** [Koenkk/zigbee2mqtt#11350] [Devices]
I Have [Xiaomi RTCGQ11LM](https://www.zigbee2mqtt.io/devices/RTCGQ11LM.html#xiaomi-rtcgq11lm) and it's reporting the wrong temperature in my room.
I'd like to disable the temperature exposure.
I'm trying with:

```
            "0x00158d0006ca4055": {
                "filtered_attributes": [
                    "temperature"
                ],
                "friendly_name": "Motion",
                "optimistic": true,
                "temperature_calibration": 0,
                "temperature_precision": 0
            },
```

But it doesn't do anything.

Using the Conbee II firmware 0x26720700

Thanks!

**Comment by ajlunis** [comment]
You ever figure this out?  I'm having the same issue.

**Reply by OmriMulti** [reply]
I disabled it in the Homebridge plugin. no success via the zigbee2mqtt config....

**Reply by Setka20** [reply]
Hi where can I disable This in homebridge. I don’t see This option
